src/dso_api/dynamic_api/serializers.py

```diff
@@ -425,6 +425,74 @@ class DynamicLinksSerializer(DynamicSerializer):
 
     serializer_related_field = HALTemporalHyperlinkedRelatedField
 
+    def get_fields(self):
+        """Avoid adding fields that loop back to a parent."""
+        fields = super().get_fields()
+
+        if self.parent_source_fields:
+            for name, field in fields.copy().items():
+                if self._is_repeated_relation(name, field):
+                    logger.debug(
+                        "Excluded %s from _links field, it resolves to a parent via %s.%s",
+                        name,
+                        ".".join(f.name for f in self.parent_source_fields),
+                        field.source,
+                    )
```

Comment:
The string join is executed even if the logger is set to a higher than `DEBUG`.
```suggestion
                    if logger.isEnabledFor(logging.DEBUG):
                        logger.debug(
                            "Excluded %s from _links field, it resolves to a parent via %s.%s",
                            name,
                            ".".join(f.name for f in self.parent_source_fields),
                            field.source,
                        )
```

Reply:
Adressed in PR #366 :-)